### ISSUE TITLE
Bed outline and grid is gone from the Cura v.5.2.1 PREPARE and PREVIEW screens

### ISSUE THREAD

**Issue by yego666** [Ultimaker/Cura#14374]
### Application Version

5.2.1

### Platform

Windows 10

### Printer

TronXY X5SA Pro 

### Reproduction steps

1. opened Cura for the 777th time expecting to work as usual, i,e. without any flaws.
2. Nonew software or settings were made on the notebook between the last and the current session. Nothing has changed in terms of OS
3. Application opened without problems, but the bed outline and the grid are missing on both PREVIEW and PREPARE views.
4. Models can be loaded and repositioned as before, but no trace of my bed any more.
5. Oddly, the TRONXY logo is still displayed where it used to be all the time.
6. The same after several machine shutdowns and reboots.
7. All functionality is still working and sliced models can be properly printed, yet the bed is missing all the time.
8. I wouldn't have complained if it had not been difficult to position several models at the same bed.
9. I haven't tried reinstallation yet. Perhaps there is an easier way to fix the problem.

### Actual results

No positive results.
Bed is still missing :(

### Expected results

The problem should be cured.

### Checklist of files to include

- [X] Log file
- [ ] Project file

### Additional information & file uploads

[cura.log](https://github.com/Ultimaker/Cura/files/10490839/cura.log)

**Comment by MariMakes** [comment]
Hey @yego666,

Welcome to the Ultimaker Cura Github 🚀 
Sorry, it took us a while to get back to you 😞 

Do you have a screenshot for us? The grid shows up for me
![image](https://user-images.githubusercontent.com/40423138/216610379-f387e5fa-2b68-4fe2-b7db-d746331d1c17.png)

I don't see anything odd in the logs, I do see that you have a number of plug-ins installed.
Can you to Marketplace and click on the gear in the top right corner?
You can disable some of the plug-ins to see if one of them might be affecting your preview. 

The last thing you can check is in the Preferences | Configure Cura, to see if you have compatibility mode enabled. 
That also influences your preview screen. 

Can you let us know if any of these tips work, or provide us with a screenshot?
Thank you 🙏

**Comment by yego666** [comment]
Hey.
Thank you very much for your reply.
You did a great job creating such a complicated tool like Cura. I really do
appreciate your job. I realize that no software is flaw free, so I tried
all things hinted in your email some time ago. Unfortunately none of them
worked for me.
I guess that some of MS updates must have messed up something, so there is
no real chance to spot the culprit.
Finally I decided to reinstall the slicer and miraculously it cured the
problem.
I am sorry for not waiting long enough for your response, but I needed
my/Your Cura badly.
Nevertheless I thank you very much for your support and constant
improvements which make 3D printing easier and ever more interesting.
Best regards,
Mark
